### DISCUSSION TITLE
What Is a False Positive and False Negative and How Are They Significant?

### DISCUSSION THREAD

**Discussion by luffy-Monkey123** [luffy-Monkey123/python-cool-programs#18] [Q&A] [answered]
What Is a False Positive and False Negative and How Are They Significant?

**Comment by RUTAGENGWA-ASANTE-BRUCE** [comment, marked as answer]
False positives are those cases that wrongly get classified as True but are False. 

False negatives are those cases that wrongly get classified as False but are True.

In the term ‘False Positive,’ the word ‘Positive’ refers to the ‘Yes’ row of the predicted value in the confusion matrix. The complete term indicates that the system has predicted it as a positive, but the actual value is negative.
